toggl.sh.example

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Use by command:
+# $ ./toggl.sh 09.22.2017
+# or simply (current date will be used)
+# $ ./toggl.sh
+
+TOGGLE_API_KEY="<toggle_api_key_here>"
+REDMINE_URL="<redmine_url_here>"
+REDMINE_API_KEY="<redmine_api_key_here>"
+
+# date format = mm.dd.YYYY
```

Comment:
What do you think about replacing this with `$(date -d "$1" "+%m.%d.%Y")'`?

You could use those arguments then:
- yesterday
- -3 days
- dd.mm.yyyy
- etc.

Reply:
Unfortunately it wont work on osx, where syntax of `date` command is different.